docs/writing-tests/interaction-testing.mdx

```diff
@@ -56,24 +56,71 @@ Once the story loads in the UI, it simulates the user's behavior and verifies th
 
 <Video src="../_assets/writing-tests/addon-interaction-example-optimized.mp4" />
 
-### Run code before each test
+<If notRenderer="angular">
 
-It can be helpful to run code before each test to set up the initial state of the component or reset the state of modules. You can do this by adding an asynchronous `beforeEach` function to the story, meta (which will run before each story in the file), or the preview file (`.storybook/preview.js|ts`, which will run before every story in the project).
+### Run code before the component gets rendered
 
-Additionally, if you return a cleanup function from the `beforeEach` function, it will run **after** each test, when the story is remounted or navigated away from.
+You can execute code before rendering by using the `mount` function in the `play` method.
+
+Here's an example of using the [`mockdate`](https://github.com/boblauer/MockDate) package to mock the [`Date`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date), a useful way to make your story render in a consistent state.
+
+<CodeSnippets path="mount-basic.md" />
+
+<Callout variant="warning">
+  There are two requirements to use the `mount` function:
+
+  1. You *must* destructure the mount property from the `context` (the argument passed to your play function). This makes sure that Storybook does not start rendering the story before the play function begins.
+  2. Your Storybook framework or builder must be configured to transpile to ES2017 or newer. This is because destructuring statements and async/await usages are otherwise transpiled away, which prevents Storybook from recognizing your usage of `mount`.
+</Callout>
+
+<If renderer={['react', 'vue', 'svelte']}>
```

Comment:
Given that the snippets are only available for these renderers, it seemed like the feature itself only worked for these, so I conditionally rendered this section.

Is there appropriate fallback content for the others? @kasperpeulen

Reply:
The fallback is kind of the `basic-mount` example, which will be displayed above this, so I think this is fine!